lms/resources/_js_config/__init__.py

```diff
@@ -51,7 +51,6 @@ def add_document_url(self, document_url):
         if document_url.startswith("blackboard://"):
             self._config["api"]["viaUrl"] = {
                 "authUrl": self._request.route_url("blackboard_api.oauth.authorize"),
-                "product": self._request.product.family,
```

Comment:
This was completely wrong. It was not in the right place it and, in any case this are `GET` request so the mechanism was wrong.